src/Components/NavLinks/index.jsx

```diff
@@ -0,0 +1,15 @@
+import { Link } from "react-router-dom"
+
+function NavLinks(modalOpen, handleClick) {
+    return(
+        <div>
+            {modalOpen && (
```

Comment:
Once you've done the other two comments, you should be able to remove this && entirely - whether or not the modal is displayed will be entirely controlled by the Nav component

Reply:
Note: This will mean that the modalOpen prop is no longer needed - so you can remove that from this function